src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -500,29 +576,36 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <Target Name="_ComputeUseBuildDependencyFile"
```

Comment:
Do we have a scenario for excluding assets from the publish output but not the build output?  Can we just have the same knobs control both, and get rid of this target and a lot more of the differences between build and publish?

Reply:
Yes, currently Razor projects have a "design" package that is `PrivateAssets="all"`.  See my general comment regarding why publish and build deps generation isn't completely unified.